pages/_app.js

```diff
@@ -19,6 +20,7 @@ import alertBannerData, { ALERT_BANNER_ACTIVE } from 'data/alert-banner'
 NProgress({ Router })
 // const { ConsentManager, openConsentManager } = createConsentManager({
 //   preset: 'oss',
+//   otherServices: [...localConsentManagerServices]
 // })
```

Comment:
Hey @dstaley , was there a reason the consent manager for Terraform was disabled? I'll leave it be if so!

Reply:
It was originally disabled to give the TF team time to prepare for their analytics to get all wonky. Feel free to turn it on now that we're past that point!